ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -756,6 +757,10 @@ ethernet_interfaces:
     isis_network_point_to_point: < boolean >
     ptp:
       enable: < true | false >
+    service_profile: < qos_profile >
+    qos_trust: < dscp defualt(L3) | cos default(L2) >
```

Comment:
typo, def**au**lt(L3)

Reply:
These defaults are not AVD defaults are they? We do not need to describe EOS defaults here.